templates/projects/empty/project.json

```diff
@@ -3,9 +3,9 @@
   "version": "1.0.0-*",
 
   "dependencies": {
-    "Microsoft.AspNet.Server.IIS": "1.0.0-beta5",
-    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta5",
-    "Kestrel": "1.0.0-beta5"
+    "Microsoft.AspNet.Server.IIS": "1.0.0-beta6",
+    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta6",
+    "Kestrel": "1.0.0-*"
```

Comment:
If the .zip file has * then its wrong. We should specify the version explicitly.